### DISCUSSION TITLE
所有功能失效

### DISCUSSION THREAD

**Discussion by PIndividual** [the1812/Bilibili-Evolved#3198] [提问 / Q&A]
如图，我安装了不少功能，但是不知何时起全部失效无法使用了，仅保留的下载封面还是灰的
<img width="263" alt="image" src="https://user-images.githubusercontent.com/62368342/160640171-a2176130-a25e-471c-955a-02d6e4a9774b.png">
<img width="302" alt="image" src="https://user-images.githubusercontent.com/62368342/160640247-2fbd17ef-5d3b-40ee-ace4-971dc827522e.png">

**Comment by Geeyun-JY3** [comment]
#3191

**Reply by PIndividual** [reply]
这个目前有解吗？

**Reply by legend418** [reply]
B站更新了一波，等更新吧

**Reply by Geeyun-JY3** [reply]
发错了，应该是 #3187

我的意思是合并重复。
至于有无解决方法，请关注 #3187 后续动向。
